### ISSUE TITLE
After the home page was created, i checked the accessibility colour needs of the page, and there was an issue with colour that was chosen while hovering on the buttons

### ISSUE THREAD

**Issue by manni8436** [manni8436/personal-porfolio#10]

**Comment by manni8436** [comment]
The initial check showed that it failed almost all sections, the colour of the text had to be changed while on hover.


![Image](https://github.com/manni8436/personal-porfolio/assets/79054812/e6220031-e2ae-44ad-88cf-c0ec037cbf56)

After changing the colour while on hover from having the text colour **white**, I changed it to showing **black** while and as a result, it improved the accessibility results.

![Image](https://github.com/manni8436/personal-porfolio/assets/79054812/f41c2760-2eab-46b5-b593-2915193c8c66)
